temoa/extensions/growth_rates/components/growth_capacity.py

```
@@ -0,0 +1,126 @@
+) -> set[tuple[Region, Period, Technology, str]]:
+    return {
+        (r, p, t, op)
+        for r, t, op in model.limit_degrowth_capacity.sparse_keys()
+        for p in model.time_optimize
+    }
+
+
+def limit_growth_capacity_constraint_rule(
+    model: GrowthRatesModel, r: Region, p: Period, t: Technology, op: str
+) -> ExprLike:
+    return limit_growth_capacity(model, r, p, t, op, False)
```

Ruff (0.15.21)

[warning] Boolean positional value in function call

(FBT003)